examples/image_reconstruction.ipynb

```diff
@@ -0,0 +1,705 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "4rGqr2VSyqrK",
+    "outputId": "2537ca9d-ee9f-4aeb-e91d-e148e5827e3f"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: poutyne in /usr/local/lib/python3.6/dist-packages (1.2)\n",
+      "Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from poutyne) (1.19.5)\n",
+      "Requirement already satisfied: torch in /usr/local/lib/python3.6/dist-packages (from poutyne) (1.7.0+cu101)\n",
+      "Requirement already satisfied: dataclasses in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (0.8)\n",
+      "Requirement already satisfied: future in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (0.16.0)\n",
+      "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.6/dist-packages (from torch->poutyne) (3.7.4.3)\n"
+     ]
+    }
+   ],
+   "source": [
+    "pip install poutyne"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "Uwt_Nl1Qw_Pz"
+   },
+   "outputs": [],
+   "source": [
+    "#importing the needed libraries\n",
```

Comment:
Maybe put it as text before the code snippet.

For [example](https://poutyne.org/examples/introduction.html#introduction-to-pytorch-and-poutyne) something like "Let’s import all the needed packages." as we say in other example.